src/guacscanner/guacscanner.py

```diff
@@ -575,18 +624,17 @@ def process_instance(
     entity_id,
 ):
     """Add/remove connections for the specified EC2 instance."""
-    logging.debug("Examining instance %s.", instance.id)
+    instance_id = instance.id
+    logging.debug("Examining instance %s.", instance_id)
```

Comment:
Thoughts on bumping the minimum supported Python version to `3.8` and enabling the glorious
![image](https://user-images.githubusercontent.com/50747025/152363514-337337c7-c413-4867-8b69-5bb308a552d9.png)
?

```suggestion
    logging.debug("Examining instance %s.", (instance_id := instance.id))
```